cost-analyzer/values-eks-cost-monitoring.yaml

```diff
@@ -67,8 +67,22 @@ kubecostModel:
     #  cpu: "800m"
     #  memory: "256Mi"
 
+serviceAccount:
+  create: true # Set this to false if you're bringing your own service account.
+  annotations: {}
+  # name: kc-test
+
+# Define persistence volume for cost-analyzer
+persistentVolume:
+  size: 32Gi
+  dbSize: 32.0Gi
+  enabled: true # Note that setting this to false means configurations will be wiped out on pod restart.
```

Comment:
I feel like this default could be impactful in the long run, especially on large clusters. What factors led us to pick 32Gi specifically?

Reply:
@michaelmdresser  the change was to match with the config of our main Values file at https://github.com/kubecost/cost-analyzer-helm-chart/blob/648d51d7d380288bc3a037029f1411575aa375de/cost-analyzer/values.yaml#L381 . If it is not a recommended size, I can change it back to "0.2 Gi"

Reply:
Thank you for helping to review Michael. About your question, I checked with Solution team and we do not see the size  of PV for Cost Analyzer is an issue for the prospects but the pv size of prometheus is more important.